### ISSUE TITLE
Revisit/update task template

### ISSUE THREAD

**Issue by anenadic** [EVERSE-ResearchSoftware/RSQKit#140]
In ./pages/your_tasks/TEMPLATE_your_tasks.md.

"Training mentioned on the page" section (currently not existing) and "Tools and resources mentioned on the page" section (currently exists) can be added automaticaly and do not need to be listed manually in the template.

We may also decide to do something else with the template and add "Indicators mentioned on the page" section too.

And make sure that existing pages align with the new task template.

See duplicate issue #161.

**Comment by shoaibsufi** [comment]
tl;dr - remove '## How to cite this page' and  '## References' from the task template


Ask part of #129 I am updating pages - I am removing the manual '## Tools and resources' section - I have noticed that we have a '## How to cite this page' and '## References' headings - two headings that the RDMkit template does not.

I think it's probably best to remove the '## How to cite this page' heading and replace it with some kind of magic auto link that perhaps the ETT could provide at some stage

Also for ''## References' not all pages have references and we could just use links to references where need be in the write up - we probably won't be referring to them an onerous number of times - also when we had information from the TF2 Reference framework into the e.g. Research Software or Research Software Quality pages as theory - that might be the place to support references - and in the task pages we will probably link to those theory pages in RSQKit also.

**Comment by dgarijo** [comment]
`How to cite this page` was one of the main things Carole insisted on having. It's great to have contributors, so credit should be given to the participants of the page. And other resources do have this kind of guide.

If references are provided explicitly inline, then ok to remove the `References` section from my end. But in some cases there are papers, giving the proper credit would be appropriate.

**Comment by anenadic** [comment]
I think the 'How to cite this page' was meant to provide an official reference to this page - the idea that Carole was talking about (also for RDMKit) was to have citations for each individual page (like mini articles) - that was the reason why we added this section. However, we do not currently have a realistic mechanism to obtain DOIs/citations for individual pages - so this section would remain empty for the time being. 

This is in addition to the list of contributors to the page, which would be listed in one of the other section on the page.

So, we can remove this section for the time being (given that it will be empty) and bring it back in the future, or keep it for future reference/work.

**Comment by dgarijo** [comment]
You don't need a DOI to support citation. We just need to figure out how to declare in page citation as a project. Example: https://training.galaxyproject.org/training-material/topics/microbiome/tutorials/metagenomics-assembly/tutorial.html#citing-this-tutorial (no DOI, but they state how to cite their work)

If someone declares how to cite them, we should add a `References` section. If this is not in the template, it should be in the guidelines, IMO
